cmd/cloudinitexecute/cloudinitexecute.go

```diff
@@ -84,6 +85,24 @@ func ApplyConsole(cfg *rancherConfig.CloudConfig) {
 			continue
 		}
 
+		targetDir := mount[1]
+		if mount[3] == "bind" {
+			fileInfo, err := os.Stat(mount[0])
+			if err != nil {
+				log.Errorf("Unable to find %s: %v", mount[0], err)
+				continue
+			}
+			if mode := fileInfo.Mode(); !mode.IsDir() {
+				targetDir = filepath.Dir(mount[1])
+			}
+		}
+		if targetDir != "" {
```

Comment:
My mistake. I guess I was looking at an older implementation.

I'm definitely open to suggestions as to where we go from here. Perhaps @joshwget can provide some input on whether util.Mount() would work these days?